pkg/console/console.go

```diff
@@ -124,13 +124,7 @@ var (
 
 	// Colorize prints message in a colorized form, dictated by the corresponding tag argument.
 	Colorize = func(tag string, data interface{}) string {
-		if isatty.IsTerminal(os.Stdout.Fd()) {
-			colorized, ok := Theme[tag]
-			if ok {
-				return colorized.SprintFunc()(data)
-			} // else: No theme found. Return as string.
-		}
-		return fmt.Sprint(data)
```

Comment:
Removing this causes problems @ebozduman - we added it on purpose to avoid printing unparsable ASCII tags.

Reply:
@harshavardhana 

Good point.
I also thought about it and googled it to see if `Stdout` could be anything other than `Terminal/Pipe/Redirection`.

All I found was that these 3 are the only possibilities; either it is a `Terminal` or it is a `Pipe/Redirection`. 

So, if we keep the check, we will repeat the same in the else part, that is `return colorized.SprintFunc(data)`, hence the code will be meaningless. That's why I dropped it.

If desired though, we can keep the `Theme[tag]` check, but since it is an internal check, I thought hat should not a concern.

Reply:
Added a check that tests if `Stdout` is a `Pipe/Redirection` and brought back the `Terminal` check per @harshavardhana's request.

Reply:
> Added a check that tests if `Stdout` is a `Pipe/Redirection` and brought back the `Terminal` check per @harshavardhana's request.

Maybe something is not pushed - I don't see this in the changes in the PR.

Reply:
Thank you Aditya.
I'll check.

Reply:
@donatello 
You were right.
I pushed the changes. PTAL again.